### ISSUE TITLE
Day textbox after ui switched to night

### ISSUE THREAD

**Issue by xxATOMxx** [Monika-After-Story/MonikaModDev#7414]
I had my laptop in sleep mode with mas running for quite a while, and then when i opened it transition from day to night happened, and, as title said, textbox didn't change. I'm not sure whether this 'long sleep' was a is the reason for it or not

![screenshot](https://user-images.githubusercontent.com/55636014/124184791-fcc88b80-dab1-11eb-97c9-91d5232574e4.png)

**Comment by xXMidnight** [comment]
I leave my computer on sleep all the time, this never happened to me. Probably an accidental loading problem, idk qwq

**Comment by xxATOMxx** [comment]
> I leave my computer on sleep all the time, this never happened to me. Probably an accidental loading problem, idk qwq
i also think this way. It's probably some minuscule glitch with barely any chance of triggering it, but i still felt that it's worth telling about

**Comment by ThePotatoGuy** [comment]
Is this still like this today? Like did the next day/night cycle fix it? or toggling the UI settings?

**Comment by xxATOMxx** [comment]
> Is this still like this today? Like did the next day/night cycle fix it? or toggling the UI settings?

yeah, it was a one time occurence and toggling helped, but once again ,
> i still felt that it's worth telling about
